authority/provisioners.go

```diff
@@ -494,9 +494,39 @@ func optionsToCertificates(p *linkedca.Provisioner) *provisioner.Options {
 			}
 		}
 	}
+	if p.Type == linkedca.Provisioner_SSHPOP {
+		for _, wh := range p.Webhooks {
+			whCert := webhookToCertificates(wh)
+			ops.SSH.Webhooks = append(ops.SSH.Webhooks, whCert)
+		}
+	} else {
+		for _, wh := range p.Webhooks {
+			whCert := webhookToCertificates(wh)
+			ops.X509.Webhooks = append(ops.X509.Webhooks, whCert)
+		}
+	}
```

Comment:
We should add SSH support for webhooks, not just to the SSHPOP provisioner

Reply:
@maraino is there ever a case when a single provisioner can sign both x509 and ssh certificates or will it always be one or the other?

Reply:
actually I moved options up out of the x509 and ssh fields so it doesn't matter 0bb0a2b3

Reply:
Most provisioners can issue both ssh and x509 provisioners. Moving the Webhooks to the top of the options can work, but for most enriching webhooks would be very type-specific. For example, an SSH webhook might add a GitHub username, which doesn't make sense in X509. And an X509 webhook, might add specific URI SAN.

At the same time, if we use the same webhooks, we will need that all webhooks know how to handle the given data and at least return `{}` if that webhook doesn't work on an SSH certificate.